server/matchmaker/pop_timer.py

```diff
@@ -66,17 +65,18 @@ def time_until_next_pop(self, num_queued: int, time_queued: float) -> float:
         total_times = sum(self.last_queue_times)
         if total_times:
             self._logger.debug(
-                "Queue rate for %s: %f/s", self.queue_name,
+                "Queue rate for %s: %f/s", self.queue.name,
                 total_players / total_times
             )
 
         # Obtained by solving $ NUM_PLAYERS = rate * time $ for time.
-        next_pop_time = config.QUEUE_POP_DESIRED_PLAYERS * total_times / total_players
+        desired_players = config.QUEUE_POP_DESIRED_MATCHES * self.queue.team_size
```

Comment:
Oops, number of players per match is team_size x2
```suggestion
        desired_players = config.QUEUE_POP_DESIRED_MATCHES * self.queue.team_size * 2
```